core/util/src/main/java/com/ku_stacks/ku_ring/util/ThrowableUtil.kt

```diff
@@ -0,0 +1,23 @@
+package com.ku_stacks.ku_ring.util
+
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.Json
+import retrofit2.HttpException
+
+fun Throwable.getHttpExceptionMessage(): String? = when(this) {
+    is HttpException -> {
+        val errorBody = response()?.errorBody()?.string()
+
+        if (!errorBody.isNullOrBlank()) {
+            Json.decodeFromString<ErrorMessage>(errorBody).resultMsg
+        } else null
+    }
+    else -> null
+}
+
+@Serializable
+private data class ErrorMessage(
+    val resultMsg: String,
```

Comment:
```suggestion
    @SerialName("resultMsg")
    val resultMessage: String,
```

혹시 이렇게 매핑하면 어떨까요...? 축약어로 되어있어서 가독성이 살짝 떨어지네요 ㅠ

Reply:
저도 json 네이밍이 맘에 안들면 이렇게 하는 편입니다

Reply:
동의합니다! 바로 고칠게요~